ggml.c

```diff
@@ -19981,7 +19981,7 @@ int ggml_cpu_has_blas(void) {
 
 int ggml_cpu_has_cublas(void) {
 #if defined(GGML_USE_CUBLAS)
-    return 1;
+    return ggml_cublas_loaded();
```

Comment:
This won't work until ggml is initialized, and it breaks the `llama-bench` backend column. I suggest moving the check to a function in `ggml-cuda` instead.

Reply:
revert back to 1